torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -84,8 +84,10 @@ def is_default_node(node, modules):
         torch.nn.LayerNorm,
         torch.nn.Dropout,
         torch.nn.PReLU,
+        torch.nn.ChannelShuffle,
         torch.nn.BatchNorm2d,
         torch.nn.BatchNorm3d,
+        torch.nn.ChannelShuffle,
```

Comment:
why do we have two channel shuffle here?

Reply:
Sorry for the typo. Now they are removed.